README.md

```diff
@@ -1,35 +1,142 @@
+
+### Usage
+The Conjur role provides a method to establish the Conjur identity of a remote node with Ansible.
+The node can then be granted least-privilege access to retrieve the secrets it needs in a secure
+manner.
 
 ### Role Variables
 
-None.
-<br>
-<br>
+* `conjur_appliance_url` (_Optional)_: URL of the running Conjur service
+* `conjur_account` (_Optional)_: Conjur account name
+* `conjur_host_factory_token` (_Optional)_: [Host
+  Factory](https://developer.conjur.net/reference/services/host_factory/) token for layer
+  enrollment. This should be specified in the environment on the Ansible controlling host.
+* `conjur_host_name` (_Optional)_: Name of the host to be created.
+* `conjur_ssl_certificate`: Public SSL certificate of the Conjur endpoint
+* `conjur_validate_certs`: Boolean value to indicate if the Conjur endpoint should validate
+  certificates
+* `summon.version`: version of Summon to install. Default is `0.8.2`.
+* `summon_conjur.version`: version of Summon-Conjur provider to install. Default is `0.5.3`.
+
+The variables marked with _`(Optional)`_ are required fields. All other variables are required for
+running with an HTTPS Conjur endpoint.
+
+### Example Playbook
+
+Configure a remote node with a Conjur identity and Summon:
+```yml
+- hosts: servers
+  roles:
+    - role: cyberark.conjur-host-identity
+      conjur_appliance_url: 'https://conjur.myorg.com/api',
+      conjur_account: 'myorg',
+      conjur_host_factory_token: "{{lookup('env', 'HFTOKEN')}}",
+      conjur_host_name: "{{inventory_hostname}}"
+```
+
+This example:
+- Registers the host `{insert whatever the hostname is}` with Conjur, adding it into the Conjur 
+policy layer defined for the provided host factory token.
+- Installs Summon with the Summon Conjur provider for secret retrieval from Conjur.
+
+### Summon & Service Managers
+With Summon installed, using Conjur with a Service Manager (like SystemD) becomes a snap.  Here's a
+simple example of a SystemD file connecting to Conjur:
+```ini
+[Unit]
+Description=DemoApp
+After=network-online.target
+
+[Service]
+User=DemoUser
+#Environment=CONJUR_MAJOR_VERSION=4
+ExecStart=/usr/local/bin/summon --yaml 'DB_PASSWORD: !var staging/demoapp/database/password' /usr/local/bin/myapp
+```
+> Note: When connecting to Conjur 4 (Conjur Enterprise), Summon requires the environment variable
+`CONJUR_MAJOR_VERSION` set to `4`. You can provide it by uncommenting the relevant line above.
+
+The above example uses Summon to retrieve the password stored in `staging/myapp/database/password`,
+set it to an environment variable `DB_PASSWORD`, and provide it to the demo application process.
+Using Summon, the secret is kept off disk. If the service is restarted, Summon retrieves the
+password as the application is started.
 
-## Plugins
+### Recommendations
 
-### conjur_variable Lookup Plugin
+- Add `no_log: true` to each play that uses sensitive data, otherwise that data can be printed to
+  the logs.
```

Comment:
is there a follow-up issue to review the logs before the next tag?

Reply:
I will file one

Reply:
Filed: https://github.com/cyberark/ansible-conjur-collection/issues/45